.tool-versions

```diff
@@ -1,7 +1,8 @@
-azure-cli 2.37.0
+azure-cli 2.51.0
 kubectl 1.27.3
 nodejs 18.1.0
 postgres 13.5
 ruby 3.2.2
 terraform 1.5.0
 yarn 1.22.19
+a
```

Comment:
Looks like a typo was added accidentally.

Reply:
Should be fixed now